main.go

```diff
@@ -318,6 +334,21 @@ func dataFromSQL(driver string, config spanner.BatchWriterConfig, client *sp.Cli
 	return writer, nil
 }
 
+func schemaFromDynamoDB(t string, sampleSize int64) (*internal.Conv, error) {
```

Comment:
Maybe rename: t --> tables.
For consistency e.g. with schemaConv.

I know I have some single letter function args, but in retrospect that is probably a mistake.

My understanding is that function args should be somewhat descriptive (but brief).